examples/table/table.bal

```diff
@@ -8,37 +8,36 @@ type Employee record {
     float salary;
 };
 
-type EmployeeSalary record {
-    int id;
-    float salary;
-};
-
 public function main() {
     // This creates an in-memory `table` constrained by the `Employee` type with the `id` marked as the
     // primary key in the column descriptor. Three data records are inserted into the `table`. The order of
     // the data values should match the order of the column descriptor.
     table<Employee> tbEmployee = table {
-        { key id, name, salary },
-        [ { 1, "Mary",  300.5 },
-          { 2, "John",  200.5 },
-          { 3, "Jim", 330.5 }
+        {key id, name, salary},
+        [
+            { 1, "Mary",  300.5},
```

Comment:
```suggestion
            {1, "Mary",  300.5},
```

Reply:
Created issue https://github.com/ballerina-platform/ballerina-lang/issues/19595 to track this